larray/inout/excel.py

```diff
@@ -22,7 +23,6 @@
 __all__ = ['read_excel']
 
 
-# We use "# doctest: +SKIP" for all tests because they work only if xlrd (an *optional* dependency) is installed
```

Comment:
the doctest +SKIP should stay, for the same reason (openpyxl is also an optional dependency)

Reply:
But openpyxl is in .travis.yml ?

Reply:
No idea but I guess so.

Reply:
FWIW, I reverted the remove of the comment above + the remove of the "# doctest: SKIP" directives.

My question is: Since an optional dependency is actually installed when Travis creates the test environment, why do we have to keep the "# doctest: SKIP" directive ?